.github/workflows/tests.yml

```diff
@@ -0,0 +1,46 @@
+name: Unit tests
+
+on:
+    push:
+      branches:
+        - master
+  
+    pull_request:
+      branches:
+        - master
+
+concurrency:
```

Comment:
It is not a concurrency, this thing cancels running jobs for the same workflow and branch. It is usually added by default  to prevent running useless jobs in case of multiple commits